mlflow/gateway/schemas/chat.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+
+from pydantic import BaseModel, Extra, Field
+
+from .common import Metadata
+
+
+class Message(BaseModel):
+    role: str
+    content: str
+
+
+class Request(BaseModel, extra=Extra.allow):
+    messages: List[Message] = Field(..., min_items=1)
+
+
+class Candidate(BaseModel):
+    message: Message
+    metadata: Metadata
+
+
+class Response(BaseModel, extra=Extra.allow):
+    candidates: List[Candidate]
+    metadata: Metadata
```

Comment:
@dbczumar Is metadata an optional field?

Reply:
I think for these providers that we're going to have to allow extras for probably all fields in the responses received. 
Anthropic's API docs: https://docs.anthropic.com/claude/reference/versioning 
Seems like they're using metadata (at the moment) for just a user uuid, but that could be added to without bumping the endpoint version according to their docs.

Reply:
Got it, will update.

Reply:
Done!